css-responsive-layout/styles.css

```
@@ -0,0 +1,49 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.fighter {
+  width: 100%;
+}
+
+.container {
+  max-width: 992px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col,
+.col-sm-half,
+.col-lg-third,
+.col-lg-fourth {
+  padding: 0 0.5rem;
+  flex-basis: 100%;
+
+  @media (width >= 576px) {
+    &.col-sm-half {
+      flex-basis: 50%;
+    }
+  }
+
+  @media (width >= 768px) {
+    &.col-lg-third {
+      flex-basis: calc(100% / 3);
+    }
+  }
+
+  @media (width >= 768px) {
+    &.col-lg-fourth {
+      flex-basis: 25%;
+    }
+  }
```

Rather than creating a whole different media query I would combine these into the same.

```suggestion
  @media (width >= 768px) {
    &.col-lg-third {
      flex-basis: (calc(100% / 3));
    }
    
    &.col-lg-fourth {
      flex-basis: calc(100% / 4);
    }
  }
```